dag-universities-b.py

```diff
@@ -1,26 +1,71 @@
 import logging as log
+from datetime import timedelta, datetime
+import os
+
 from airflow import DAG
-from datetime import datetime
 from airflow.operators.dummy import DummyOperator
+from airflow.operators.python import PythonOperator
+import pandas as pd
+import sqlalchemy
+from decouple import config
 
+#Establecemos la ruta al directorio local
+dir = os.path.dirname(__file__)
 
+#Se configura el formato de logging.ERROR
 log.basicConfig(level=log.ERROR,
                 format='%(asctime)s - %(processName)s - %(message)s',
                 datefmt='%Y-%m-%d')
 
+#Se configura la cantidad de reintentos en caso de que el DAG falle
 default_args = {
     'retries': 1,
     'retry_delay': timedelta(minutes=5),
 }
 
+# Parámetros de la base de datos
+DB_USER = config("DB_USER")
+DB_PASSWORD = config("DB_PASSWORD")
+DB_HOST = config("DB_HOST")
+DB_PORT = config("DB_PORT")
+DB_NAME = config("DB_NAME")
+
+#Se coonfigura la conexión con la base de datos
+try:
+    engine = sqlalchemy.create_engine(
+    f'postgresql+psycopg2://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}')
+    log.info('Conexión exitosa con la base de datos')
+except Exception as e:
+        log.error(e)
+
+def sql_query_extract():
+    '''
+    Lee la SQL query para las universidades B
+    Crea un DataFrame con Pandas
+    Exporta el df a un archivo csv dentro de la carpeta files
+    '''
+    try:
+        query = open(f'{dir}/sql/universidades-b.sql', 'r')
+        df = pd.read_sql_query(query.read(), con=engine)
+        # Creo carpeta files si no existe
+        if not os.path.exists(f"{dir}/files"):
+            os.makedirs(f"{dir}/files")

```

Comment:
La validación es necesaria, pero no es necesario hacerlo con un if. Te dejo un detalle sobre la función makedirs que podes aprovechar.

Syntax: os.makedirs(path, mode = 0o777, exist_ok = False) 
Parameter: 
path: A path-like object representing a file system path. A path-like object is either a string or bytes object representing a path.
mode (optional) : A Integer value representing mode of the newly created directory..If this parameter is omitted then the default value Oo777 is used.
exist_ok (optional) : A default value False is used for this parameter. If the target directory already exists an OSError is raised if its value is False otherwise not. For value True leaves directory unaltered. 
Return Type: This method does not return any value.